conf/yarn/application.conf

```diff
@@ -22,15 +22,18 @@ gobblin.yarn.app.name=GobblinYarn
 gobblin.yarn.app.master.memory.mbs=256
 gobblin.yarn.initial.containers=2
 gobblin.yarn.container.memory.mbs=512
-gobblin.yarn.conf.dir=<directory where Gobblin on Yarn related configuration files are located>
-gobblin.yarn.lib.jars.dir=<directory where Gobblin on Yarn lib jars are located>
-gobblin.yarn.app.master.files.local=${gobblin.yarn.conf.dir}"/log4j-yarn.properties,"${gobblin.yarn.conf.dir}"/application.conf,"${gobblin.yarn.conf.dir}"/reference.conf"
+gobblin.yarn.conf.dir=/tools/gobblin-dist/conf/yarn/
```

Comment:
Why this is being hard coded ?

Reply:
this is missed, let me change this to `gobblin.yarn.conf.dir=${GOBBLIN_HOME}/conf/yarn/` thanks for catching this, this was my local config.